grid-proxy/docs/swagger.json

```diff
@@ -44,6 +44,35 @@
                         "name": "ret_count",
                         "in": "query"
                     },
+                    {
+                        "type": "boolean",
+                        "description": "Get random patch of contracts",
+                        "name": "randomize",
+                        "in": "query"
+                    },
+                    {
+                        "enum": [
+                            "twin_id",
+                            "node_id",
+                            "type",
+                            "status",
+                            "..."
```

Comment:
what does the `...` mean ? and if it means any why do we use enum in the first place. I think it's better if we specify a static list of possible sort fields instead of allow of passing any value that might not be defined